src/components/file/_create.md

```diff
@@ -53,10 +53,9 @@ const [file, setFile] = React.useState();
     branch='testing'
```

Comment:
You commented out the testing branch in one place, can you also do it here? A pain when I have to keep remembering to change this to master or remove it.